### PR TITLE
support bitbucket!

### PR DESCRIPTION
**Pull request by laixintao**
Hi @ruanyl 

Thank you for your work! You saved my life !

I suffered so much to find code link in bitbucket, using your code I just made some small changes then I can use it in bitbucket, awesome!

For not breaking your old code, I just copy and make it fit bitbucket url, code tested in Vim8.

### CODE REVIEW

**Comment by ruanyl** [comment]
@laixintao Thanks for the PR 👍 
What changes you did to the function in order to make it work with bitbucket? I see you changed the `sed` command a bit, what else?

File: plugin/vim-gh-line.vim [hunk 1]
```diff
@@ -73,13 +81,64 @@ func! s:gh_line() range
...
+
+    " Set Line Number/s
+    if a:firstline == a:lastline
+        let lineRange = '-' . lineNum
+    else
+        let lineRange = 'L' . a:firstline . "-L" . a:lastline
+    endif
+
+    " String Setup
+    let src = "/src/"
```
Comment (laixintao) [inline plugin/vim-gh-line.vim]: blob -> src

File: plugin/vim-gh-line.vim [hunk 2]
```diff
@@ -73,13 +81,64 @@ func! s:gh_line() range
...
+    let sed_cmd = "sed 's\/git@\/https:\\/\\/\/g; s\/.git$\/\/g; s\/\.org:\/.org\\/\/g'"
+
+    " Get Directory & File Names
+    let fullPath = resolve(expand("%:p"))
+    let fileDir = resolve(expand("%:p:h"))
+    let cdDir = "cd " . fileDir . "; "
+    let fileName = resolve(expand('%:t'))
```
Comment (laixintao) [inline plugin/vim-gh-line.vim]: bitbucket's hashtag on url  #setup.py-21 , so I need filename

File: plugin/vim-gh-line.vim [hunk 3]
```diff
@@ -73,13 +81,64 @@ func! s:gh_line() range
...
+    let origin = <SID>StripNL(origin)
+    let commit = <SID>StripNL(commit)
+    let gitRoot = <SID>StripNL(gitRoot)
+    let fullPath = <SID>StripNL(fullPath)
+
+    " Git Relative Path
+    let relative = split(fullPath, gitRoot)[-1]
+
+    " Form URL With Line Range
+    let url = origin . src . commit . relative . '#' . fileName . lineRange
```
Comment (laixintao) [inline plugin/vim-gh-line.vim]: and here

**Comment by laixintao** [comment]
@ruanyl I think that's all, you can refactor the code then I will test it on bitbucket again.

**Comment by ruanyl** [comment]
merged with a small refactor, thanks!

**Pull request merged**